api/loader/fileloader_test.go

```diff
@@ -635,6 +661,8 @@ func TestLoaderHTTP(t *testing.T) {
 
 // setupOnDisk sets up a file system on disk and directory that is cleaned after
 // test completion.
+// TODO(annasong): Move all loader tests that require real file system into
```

Comment:
Looks like, according to your comment, you have a plan to remove the [setupOnDisk()](https://github.com/kubernetes-sigs/kustomize/pull/5225/files#diff-f036a91524edc0f1e5eae1765348f9b6a9d79bfa43200296010714ce1a4e97f2R666) function and replace it with the [Setup()](https://github.com/kubernetes-sigs/kustomize/pull/5225/files#diff-519525fb0fa60ef37894f72521cab550570a1752e06c09d32828e839bbbcbe45R16) function?

Reply:
Essentially, yeah. I plan to move all tests calling `setupOnDisk` into `krusty/remoteloader_test.go` (will rename to `loader_test.go`), then replace calls to `setupOnDisk` with `Setup`. 

I can't replace `setupOnDisk` with `Setup` in `fileloader_test.go` because it would create a cyclic dependency.

Reply:
That is a great plan!